src/public/js/user.js

```diff
@@ -79,3 +79,17 @@ function confirmDeletion(message) {
   var message = $('#confirmation-message').text();
   return confirm(message);
 }
+
+$(document).ready(function () {
```

Comment:
chỉ cần 1 block này trong file là đc e, toàn bộ code bên trong em move lên block phía trên nhé